test/test_operations.py

```diff
@@ -722,6 +722,60 @@ def test_frac_negative(self):
     xla_b = xla_a.frac()
     self.assertEqual(b, xla_b)
 
+  def test_flip(self):
```

Comment:
Can we split this up into tests which are testing individual functionalities (with proper test suffix name - test_flip_+SUFFIX)?